docs/drill4net/support-matrix.mdx

```diff
@@ -0,0 +1,80 @@
+﻿---
+id: support-matrix
+title: Support Matrix
+---
+import { Image } from '@components';
+import { Info } from '@components';
+import { Table } from '../../src/components'
+
+## Testing Frameworks
+
+<Table>
+
+| Framework |      Supported versions     |
+|:---------:|:---------------------------:|
+| [NUnit](https://nunit.org)     |	      3.13.3						|
+| [XUnit](https://xunit.net)     | 2.4.2, 2.5.0, 2.5.1, 2.5.2, 2.5.3	|
+| [Selenium](https://www.selenium.dev)  |	4.8.1, 4.12.2, 4.14.1		|
```

Comment:
Change link to point to the specific component drill4net instruments/patches (I suspect it is the Selenium Driver for .NET)

Reply:
Same goes for Playwright

Reply:
For Selenium and Playwright, the links changed to the package source